layouts/shortcodes/cli/influxd-flags.md

```diff
@@ -1,35 +1,39 @@
+|      | `--assets-path`                                             | Override default assets by serving from a specific directory (developer mode)                                                                                                                                                                                              | string     | `INFLUXD_ASSETS_PATH`                |
+|      | `--bolt-path`                                               | Path to boltdb database (default `~/.influxdbv2/influxd.bolt`)                                                                                                                                                                                                             | string     | `INFLUXD_BOLT_PATH`                  |
+|      | `--e2e-testing`                                             | Add /debug/flush endpoint to clear stores; used for end-to-end tests (default `false`)                                                                                                                                                                                     |            | `INFLUXD_E2E_TESTING`                |
+|      | `--engine-path`                                             | Path to persistent engine files (default `~/.influxdbv2/engine`)                                                                                                                                                                                                           | string     | `INFLUXD_ENGINE_PATH`                |
+| `-h` | `--help`                                                    | Help for the `influxd` command                                                                                                                                                                                                                                             |            |                                      |
+|      | `--http-bind-address`                                       | Bind address for the REST HTTP API (default `:8086`)                                                                                                                                                                                                                       | string     | `INFLUXD_HTTP_BIND_ADDRESS`          |
+|      | `--influxql-max-select-buckets`                             | Maximum number of group by time bucket a SELECT can create. A value of zero will max the maximum number of buckets unlimited.                                                                                                                                              | integer    |                                      |
```

Comment:
group by time buckets (plural)?   
(maybe) that a SELECT statement (or query)?

will max the maximum number of buckets unlimited. >> allows an unlimited number of buckets.